README.md

```diff
@@ -207,22 +207,22 @@ By default, all possible comparisons are performed. Use `--deg` to change this.
 
 #### Read files (required)
 
-Specify your read files in `FASTQ` format with `--reads input.csv`. The file `input.csv` has to look like this for single-end reads:
+Specify your read files in `FASTQ` format with `--reads input.csv`. The file `input.csv` has to look like this for single-end reads (just leave R2 empty):
 
 ```csv
-Sample,R,Condition,Source
-mock_rep1,/path/to/reads/mock1.fastq.gz,mock,
-mock_rep2,/path/to/reads/mock2.fastq.gz,mock,
-mock_rep3,/path/to/reads/mock3.fastq.gz,mock,
-treated_rep1,/path/to/reads/treat1.fastq.gz,treated,
-treated_rep2,/path/to/reads/treat2.fastq.gz,treated,
-treated_rep3,/path/to/reads/treat3.fastq.gz,treated,
+Sample,R1,R2,Condition,Source,Strandedness
```

Comment:
We should provide an example. Users will not know what to enter as Strandedness values. Maybe we even write in the Readme smt like:

* 0 = non strand-specifc
* 1 = stranded-specific (positive sense, which means in the sense of the annotated genes)
* 2 = strand-specific (reverse stranded, which means reads are mapping on the opposite strand of an annotated gene)

Reply:
Okay I'll add something like we already have in the [help message](https://github.com/hoelzer-lab/rnaflow/blob/read_auto_detection/README.md?plain=1#L304) then.